ports/croncpp/portfile.cmake

```
@@ -0,0 +1,31 @@
+
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO mariusbancila/croncpp
+  REF 11cce4666a06c40346c7ba380ddd90c53806809d #master on 9/4/2020 
+  SHA512 8f4d892ce90d8eca3711b21728bb599bf64857b20c0b143c5277687d0b6e5d5b8bf3e6dc7f9e8d028ba4e5ee711a5a9e750bcc2f771177d2f659c0c19e12207a
+  HEAD_REF master
+  PATCHES 0001-fix-cmake.patch
+)
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    PREFER_NINJA # Disable this option if project cannot be built with Ninja
+    # OPTIONS -DUSE_THIS_IN_ALL_BUILDS=1 -DUSE_THIS_TOO=2
+    # OPTIONS_RELEASE -DOPTIMIZE=1
+    # OPTIONS_DEBUG -DDEBUGGABLE=1
+)
+
+vcpkg_install_cmake()
+
+# # Moves all .cmake files from /debug/share/croncpp/ to /share/croncpp/
+# # See /docs/maintainers/vcpkg_fixup_cmake_targets.md for more details
+#vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/croncpp)
```

```suggestion
vcpkg_fixup_cmake_targets()
```
This directive is still needed to fuse the release and debug config files together.